src/Core/src/Exception/Binder/DuplicateBindingException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Spiral\Core\Exception\Binder;
+
+use Spiral\Core\Exception\ConfiguratorException;
+
+final class DuplicateBindingException extends ConfiguratorException
```

Comment:
Please describe the solution with `removeBinding` in the phpdoc

Reply:
Added information about removeBinding